src/ipc-handlers.ts

```diff
@@ -208,6 +213,11 @@ export async function createSite(
 		}
 	}
 
+	// If not using the latest WordPress version, copy our mu-plugin to disable auto-updates
+	if ( wpVersion?.version && ! wpVersion.isLatest ) {
+		await server.addMuPlugin( STUDIO_DISABLE_WP_AUTO_UPDATES_PLUGIN );
```

Comment:
As I mentioned in the issue, this might not be an effective check. Today I can create a site with the latest version, and tomorrow a new WP version may be published, making my version no longer the latest.

Reply:
I proposed an approach for that above. If the user keeps the latest (6.7.2 from the Always Latest group), they will have auto updates. If the user selects any other named version, like 6.7.2 below, auto-updates will be disabled.

As you pointed out, we have a code that handles that, which was likely broken at some point.

Reply:
The code we have seems to be inherited from wp-now CLI parameters, so it is not really used in Studio.<br>Having groups makes sense to me. "Always latest" could lead to confusion, as the user needs to manually update major versions, and minor updates can take time depending on whether the site is started and accessed. We could say something like "Auto updates enabled."

Reply:
> We could say something like "Auto updates enabled." Maybe moving nightly to that group also makes sense.

We could say "Auto Updated" to make it shorter, still clear, and consistent with other groups.